### ISSUE TITLE
Mini Cart does not display anything after adding products on AT sites

### ISSUE THREAD

**Issue by namiokuzono** [woocommerce/woocommerce-gutenberg-products-block#6874]
## Describe the bug

When the Mini Cart block is added to Atomic site, the floating cart area is empty. 

But this does not mention it being on Atomic site. 

## To reproduce

- Install WooCommerce Blocks on Atomic site/WPcom
- Use any Automattic premium theme or Blank Canvas theme
- Edit the site in FSE to add Mini Cart on the header
- Once added, try adding any product to the cart, then click Mini Cart to view the floating cart. 

## Expected behavior

You should be able to see a list of product being added.

## Screenshots

![https://cln.sh/v3wDoD](https://cln.sh/v3wDoD+)
Image link: https://cln.sh/v3wDoD

## Environment

Any atomic site hosted on WordPress.com

Check this staging site example: https://funwallpaperart.blog/product/woman-accesaries29/

**Comment by namiokuzono** [comment]
5466590-zd

**Comment by danielwrobert** [comment]
@namiokuzono I can’t seem to reproduce it but I do see the issue present on the site linked. I tested on [an atomic site of mine](https://atomictests.wpcomstaging.com/) using both TT2 and Kerr themes.

That said, I am getting some super odd styling going on…

Kerr theme - cart with item:
![CleanShot 2022-08-25 at 15 13 56](https://user-images.githubusercontent.com/481776/186750504-a741b5be-a60e-49e9-86c3-92f581e6b3be.png)

TT2 theme - empty cart:
![CleanShot 2022-08-25 at 15 17 10](https://user-images.githubusercontent.com/481776/186750462-015a1c17-457a-4c84-9f1b-056b3154e036.png)

We will have to investigate further and follow up!

**Comment by jamiepalatnik** [comment]
Adding that I came across another user report for this issue, but we are not able to recreate it on their site: 5532232-zd-woothemes

Editing to add that I can recreate this when unproxied. Advised user to use the default cart page instead of the mini cart block as a workaround.

**Comment by jamiepalatnik** [comment]
Another report of odd styling in the mini cart: 37393630-hc

Advised user to use the default cart page instead of the mini cart block as a workaround.

**Comment by kaushikasomaiya** [comment]
5676862-zen

Previously I had identified Page Optimize plugin was causing this issue. 

Ref: 5680237-zen

**Comment by Aljullu** [comment]
Hey there, I think this was fixed in https://github.com/woocommerce/woocommerce-blocks/pull/7794, which was released in WC Blocks 8.7.6 and WC core 7.1.1.

Do you think we can close this issue, or have any users been able to reproduce it since then?

**Comment by danielwrobert** [comment]
@Aljullu yes, it looks like this issue is resolved now.

I tested and confirmed it is no longer present on an atomic test site of mine with TT2 and Kerr themes. I also tested the site linked in the issue above (running the Kerr theme) to confirm that everything is working as expected there.

WRT the drawer styling issues I mentioned above, I was able to pinpoint the problem to the CoBlocks plugin.

With the CoBlocks plugin activated:
<img width="1281" alt="CleanShot 2023-03-14 at 15 10 25@2x" src="https://user-images.githubusercontent.com/481776/225112463-fa33c148-d510-4143-8811-f60cd608dd7f.png">

With the CoBlocks plugin disabled:
<img width="1277" alt="CleanShot 2023-03-14 at 15 11 14@2x" src="https://user-images.githubusercontent.com/481776/225112522-0c29f304-7d5c-4836-9890-6e3cb4d38947.png">

I'm going to go ahead and close this. Thanks!
